core/shared/src/main/scala/matryoshka/package.scala

```diff
@@ -35,101 +35,103 @@ import scalaz._, Scalaz._
   * @groupdesc dist Natural transformations required for generalized folds and unfolds.
   */
 package object matryoshka {
+  type Fix[F[_]] = matryoshka.data.Fix[F]
+  val Fix        = matryoshka.data.Fix
```

Comment:
Oh right, it was my local response to trying to test quasar against the plugin-updated matryoshka. Which was completely out of reach in a reasonable amount of time so I came back and added the alias. Didn't mean to push it.